Serilog.Sinks.Dynatrace.csproj

```diff
@@ -1,20 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
One question shouldn't stop your pr but.
.netstandard provides full support for the old 4.6 framework, why use multiple framework languages?

Reply:
Just following the existing code here in the PR, no opinion either way 👍